File: apenas java/extremamenteBasico.java
import java.util.Scanner;
public class extremamenteBasico {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int total = 0;
        for (int i = 0; i < 2; i++) {
            int aux = sc.nextInt();
            total += aux;
        }
        System.out.println("X = " + total);
        sc.close();
    }
}

File: apenas java/extremamenteBasico.cpp
#include <iostream>
using namespace std;
int main() {
    int total = 0;
    for (int i = 0; i < 2; i++) {
        int aux;
        cin >> aux;
        total += aux;
    }
    cout << "X = " << total << endl;
    return 0;
}